tf_quant_finance/volatility/heston_approximation_test.py

```diff
@@ -0,0 +1,154 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Heston Price method."""
+
+import numpy as np
+import tensorflow.compat.v2 as tf
+from absl.testing import parameterized
+from scipy import integrate
+from tensorflow.python.framework import test_util  # pylint: disable=g-direct-tensorflow-import
+
+import math
+import tf_quant_finance as tff
+
+
+def get_heston_prices(kappa=None,
+                     theta=None,
+                     sigma=None,
+                     rho=None,
```

Comment:
Please rename to kappas, thetas, .. for consistency

Reply:
It would be consistent with the other function, but wouldn't reflect what that function is doing though as that one is not vectorized.